data_structures/avltree.cpp

```diff
@@ -55,55 +72,73 @@ node *leftRotate(node *root) {
     return t;
 }
 
-/** Returns node with minimum value in the tree */
+/**
+ * @param root of the tree
+ * @returns node with minimum value in the tree
+ */
 node *minValue(node *root) {
-    if (root->left == NULL)
+    if (root->left == nullptr) {
         return root;
+    }
     return minValue(root->left);
 }
 
-/** Balanced Insertion */
+/**
+ * @bried inserts a new element into AVL tree
```

Comment:
brief, param[in]

Reply:
`root` is not really `@param[in]`, because it is being modified.